### PR TITLE
Some 1 7 10 mappings

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: mappings/net/minecraft/entity/Entity.mapping [hunk 1]
```diff
@@ -378,7 +378,7 @@ CLASS net/minecraft/class_864 net/minecraft/entity/Entity
 	METHOD method_2515 isSwimming ()Z
 	METHOD method_2516 getAir ()I
 	METHOD method_2517 setInLava ()V
-	METHOD method_2518 getTranslationKey ()Ljava/lang/String;
+	METHOD method_2518 getTranslatedName ()Ljava/lang/String;
```
Comment (arthurbambou) [inline mappings/net/minecraft/entity/Entity.mapping:381]: hmmm, it returns the translation key or the translated name depending on the version, what should we do?

Comment (halotroop2288) [reply mappings/net/minecraft/entity/Entity.mapping:381]: Call it getName or simply go back to having multiple branches :tiny_potato:

Comment (TheKodeToad) [reply mappings/net/minecraft/entity/Entity.mapping:381]: I think getName could work if it's not taken.

**Comment by arthurbambou** [comment]
I don't know what to do with this pr, it should probably restarted or something at this point 😔

**Pull request closed without merging**